specification/prescriptions-for-patients.yaml

```diff
@@ -129,39 +129,37 @@ paths:
                   value:
-                    $ref: "examples/GetMyPrescriptions/Bundle/success-single-exclusion.json"
+                    $ref: "examples/GetMyPrescriptions/Bundle/success/success-single-exclusion.json"
                 prescriptions-excluded:
-                  description: A successful response to a GetMyPrescriptions/Bundle request, with a multiple prescriptions excluded.
+                  description: A successful response to a GetMyPrescriptions/Bundle request, with multiple prescriptions excluded.
                   value:
-                    $ref: "examples/GetMyPrescriptions/Bundle/success-multiple-exclusion.json"
+                    $ref: "examples/GetMyPrescriptions/Bundle/success/success-multiple-exclusion.json"
                 distance-selling-pharmacy:
                   description: A successful response to a GetMyPrescriptions/Bundle request, where the performer organisation is a distance selling pharmacy.
                   value:
-                    $ref: "examples/GetMyPrescriptions/Bundle/success-distance-selling.json"
+                    $ref: "examples/GetMyPrescriptions/Bundle/success/success-distance-selling.json"
 
         "4XX":
           description: |
             An error occurred as follows:
 
             | HTTP status | Error code          | Description                                                                                                                               |
             | ----------- | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
-            | 400         | `exception`         | Missing or invalid NHS number in request                                                                                                  |
-            | 401         | `processing`        | Missing or invalid OAuth 2.0 bearer token in request                                                                                      |
-            | 404         | `not-found`         | No records found for the NHS number in the request                                                                                        |
+            | 401         | `ACCESS_DENIED`     | Missing or invalid OAuth 2.0 bearer token in request                                                                                      |
             | 408         | `timeout`           | Request timed out                                                                                                                         |
-            | 429         | `throttled`         | You have exceeded your application's [rate limit](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#rate-limits). |
+            | 429         | `throttled`         | You have exceeded your application's [rate limit]                                                                                         |
```

Comment:
why did you remove this URL?

Reply:
Must have copied and pasted it wrong when I was recovering it in an earlier commit.

Sorted now 8f22a78782e305766b504c67c5039847ae1ca26f